project/Dependencies.scala

```diff
@@ -49,6 +49,7 @@ object Dependencies {
     "org.apache.pekko" %% "pekko-connectors-cassandra" % pekkoConnectorsVersion,
     "org.apache.pekko" %% "pekko-persistence" % pekkoVersion,
     "org.apache.pekko" %% "pekko-persistence-query" % pekkoVersion,
+    "org.apache.pekko" %% "pekko-stream" % pekkoVersion,
```

Comment:
So I suspect that you originally added this because you had mismatched snapshot versions, if this is the case I suspect that if you remove this dependency it should still work/pass.

Reply:
Yes, but since the project relies heavily on streams it would make sense to me to keep the explicit dependency.

Reply:
Fair enough 👍